internal/transform/scope-html.go

```diff
@@ -95,72 +95,81 @@ func injectDefineVars(n *astro.Node, values []string) {
 
 func injectScopedClass(n *astro.Node, opts TransformOptions) {
 	hasSpreadAttr := false
-	scopedClass := fmt.Sprintf(`astro-%s`, opts.Scope)
-	for i, attr := range n.Attr {
-		if !hasSpreadAttr && attr.Type == astro.SpreadAttribute {
-			// We only handle this special case on built-in elements
-			hasSpreadAttr = !n.Component
-		}
+	if opts.ScopedStyleStrategy != "attribute" {
```

Comment:
I would like to run the equality check using the enum `ScopeStrategy.ScopeStrategyAttribute`, but I didn't find a good way to import this enum inside this file. Any help would be appreciated